styles/_base.scss

```diff
@@ -106,3 +106,14 @@ blockquote {
 iframe {
   border: none;
 }
+
+body:not(.dark) .dark-mode-only,
+.dark .light-mode-only {
+  display: none;
+}
+
+.light-mode-only,.dark-mode-only{
```

Comment:
This doesn't seem to be Stylelint conforming. There's missing spacing/padding in this whole section, and a missing file-line ending.